### PR TITLE
Add new page on updating HTTPD

### PR DESCRIPTION
**Pull request by sonotley**
We get a lot of questions about updating HTTPD.
Having a single page that summarizes this will save a lot of time.

### CODE REVIEW

**Review by djw-m — DISMISSED** [review]
LGTM

File: product_docs/docs/pem/9/upgrading/upgrading_httpd.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+The Apache HTTPD web server is used to serve the PEM web application. 
+On Linux, PEM does not bundle HTTPD, but depends on the version provided by the system package manager. 
+On Windows the PEM installer bundles a version of HTTPD.
+
+## Upgrading HTTPD on Linux
+
+!!! Important
+    If you are using Red Hat Enterprise Linux (RHEL) and seeking to update HTTPD in order to address a vulnerability,
+    please read [Note on HTTPD versioning in RHEL](#note-on-http-versioning-in-rhel), below.
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/upgrading/upgrading_httpd.mdx:16]:
```suggestion
    please read [Note on HTTPD versioning in RHEL](#note-on-httpd-versioning-in-rhel), below.
```

**Comment by nidhibhammar** [comment]
fixed the link

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**